### PR TITLE
[MODULE-2] add homework 5

### PR DESCRIPTION
**Pull request by vladimirstempel**

### CODE REVIEW

File: LoggerApp/Logger/FileService.cs [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        ProcessOldFiles(dirPath);
+    }
+
+    private void ProcessOldFiles(string dirPath)
+    {
+        if (Directory.Exists(dirPath))
+        {
+            var files = Directory.GetFiles(dirPath);
+
+            if (files.Length > 3)
```
Comment (vladboss61) [inline LoggerApp/Logger/FileService.cs:50]: if (files.Length <= 3)
{
      return;
}

File: LoggerApp/Logger/Logger.cs [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    }
+
+    public string GetLogsAsText(Level level)
+    {
+        if (Logs.TryGetValue(level, out var logs))
+        {
+            return string.Join("", logs);
+        }
+
+        return "";
```
Comment (vladboss61) [inline LoggerApp/Logger/Logger.cs:40]: return string.Empty

File: LoggerApp/Logger/Logger.cs [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+{
+    private Dictionary<Level, Collection<string>> Logs = new();
+
+    public void Write(Level logLevel, string message)
+    {
+        var stringBuilder = new StringBuilder();
+
+        stringBuilder
+            .Append($"{DateTime.Now:MM/dd/yyyy hh:mm:ss.fff tt}: ")
+            .Append($"{(Level)logLevel}: ")
```
Comment (vladboss61) [inline LoggerApp/Logger/Logger.cs:19]: use logLevel.ToString()

Comment (vladimirstempel) [reply LoggerApp/Logger/Logger.cs]: Done

File: LoggerApp/Logger/Logger.cs (hunk 2 above)
Comment (vladimirstempel) [reply LoggerApp/Logger/Logger.cs]: Done

File: LoggerApp/Logger/FileService.cs [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+            Directory.CreateDirectory(dirPath);
+        }
+
+        File.WriteAllText(filePath, logs);
+        ProcessOldFiles(dirPath);
+    }
+
+    private void ProcessOldFiles(string dirPath)
+    {
+        if (Directory.Exists(dirPath))
```
[the remark this answers is not in the document]
Comment (vladimirstempel) [reply LoggerApp/Logger/FileService.cs]: Done

**Pull request merged**